custom-bundle.Dockerfile

```diff
@@ -28,6 +28,7 @@ FROM quay.io/openstack-k8s-operators/rabbitmq-cluster-operator-bundle:latest as
 FROM quay.io/openstack-k8s-operators/placement-operator-bundle:latest as placement-bundle
 FROM quay.io/openstack-k8s-operators/glance-operator-bundle:latest as glance-bundle
 FROM quay.io/openstack-k8s-operators/cinder-operator-bundle:latest as cinder-bundle
+FROM quay.io/gibi/nova-operator-bundle:v0.0.182 as nova-bundle
```

Comment:
This is built from https://github.com/openstack-k8s-operators/nova-operator/pull/168